nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,44 @@
+Copyright 2015-2017 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Yammer Metrics
+    The following NOTICE information applies:
+      Metrics
+      Copyright 2010-2012 Coda Hale and Yammer, Inc.
+
+      This product includes software developed by Coda Hale and Yammer, Inc.
+
+      This product includes code derived from the JSR-166 project (ThreadLocalRandom), which was released
+      with the following comments:
+
+          Written by Doug Lea with assistance from members of JCP JSR-166
+          Expert Group and released to the public domain, as explained at
+          http://creativecommons.org/publicdomain/zero/1.0/
+
+************************
+Common Development and Distribution License 1.1
+************************
```

Comment:
Add this under the Apache segment:

```
    (ASLv2) Prometheus Simple Client libraries
        Copyright 2012-2019 The Prometheus Authors
```

I took that from [their NOTICE file](https://github.com/prometheus/prometheus/blob/master/NOTICE).

Then add:

```
  (ASLv2) Jetty
    The following NOTICE information applies:
       Jetty Web Container
       Copyright 1995-2017 Mort Bay Consulting Pty Ltd.

  (ASLv2) Apache Commons Codec
      The following NOTICE information applies:
        Apache Commons Codec
        Copyright 2002-2014 The Apache Software Foundation

        src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
        contains test data from http://aspell.net/test/orig/batch0.tab.
        Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)

        ===============================================================================

        The content of package org.apache.commons.codec.language.bm has been translated
        from the original php source code available at http://stevemorse.org/phoneticinfo.htm
        with permission from the original authors.
        Original source copyright:
        Copyright (c) 2008 Alexander Beider & Stephen P. Morse.

    (ASLv2) Apache Commons Lang
    The following NOTICE information applies:
      Apache Commons Lang
      Copyright 2001-2017 The Apache Software Foundation

      This product includes software from the Spring Framework,
      under the Apache License 2.0 (see: StringUtils.containsWhitespace())
```

Then add a new MIT section like this:

```
************
MIT
************

  	This product optionally depends on 'Bouncy Castle Crypto APIs' to generate
  	a temporary self-signed X.509 certificate when the JVM does not provide the
  	equivalent functionality.  It can be obtained at:

  	  * LICENSE:
  	    * license/LICENSE.bouncycastle.txt (MIT License)
  	  * HOMEPAGE:
  	    * http://www.bouncycastle.org/
```